data/schema/game_config.cfg

```diff
@@ -390,8 +427,9 @@
     [/type]
     {SUBST_TYPE coordinate}
     {SUBST_TYPE coordinates}
-    {SUBST_TYPE range_list}
+    {SUBST_TYPE positive_range_list}
     {SUBST_TYPE real_range_list}
+    {SUBST_TYPE signed_range_list}
```

Comment:
It's not like it matters, but I'd probably push this up one line to keep the subst types in the same order as the the list types.

Reply:
Can do. Not sure if it's better or worse to reorder the list types into alphabetical order instead.

Reply:
Wondering about renaming "positive_range" to "unsigned_range" - it's more accurate, but it's also visually a lot closer to "signed_range".

Reply:
I was thinking that too, but I'm not sure. The int types in schema are named `int` and `unsigned` so another possibility would be `int_range` and `unsigned_range`.